CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Developing XTF
+
+Compile from source:
+
+```mvn clean install```
+
+# XTF git development cycle <a name="devcycle"></a>
+We use the [git flow](https://guides.github.com/introduction/flow/) workflow.
+The steps are
+
+0. Read the [guide](https://guides.github.com/).
+1. Create an account [GitHub](https://github.com/).
+2. Fork the [XTF](https://github.com/xtf-cz/xtf/) repository.
+3. Clone your fork, add the upstream XTF repository as a remote, and check out locally:
+
+        git clone git@github.com:USERNAME/xtf.git
+        cd xtf
+        git remote add upstream git@github.com:xtf-cz/xtf.git
+        git branch master
+        git checkout master
+        git pull --rebase upstream master
+
+   The steps until here only need to be executed once, with the exception being the last command: rebasing against upstream master branch
+   You will need to rebase every time the upstream master is updated.
+
+4. Create a feature branch
+
+        git branch feature/BRANCH_NAME
+
+5. Make the changes to the code (usually fixing a bug or adding a feature) 
+6. **Ensure the code compiles without any error, tests pass and complies with the code style.**
+
+        mvn clean test
+        
+    If you need just recompile the code then run:
```

Comment:
`If you need to recompile the code, run:`

Did you mean `If jou need a code style chceck`? I do not understand the link with a recompiled  code and the plugin.

Reply:
fixed